frontend/src/metabase/components/LastEditInfo.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
@akiselev I'm wondering why you did not use Typescript? There's nothing that blocks it. Probably, it will require adding some `any` around redux state, but it is better than still have everything in `js`

Reply:
@ranquild also added state type so you can extend it `import { State } from “metabase-types/store”`
There are types inside `metabase-types/api`